benchmarks/__main__.py

```diff
@@ -0,0 +1,13 @@
+"""Executes the benchmarking module."""
```

Comment:
can we add the running instruction to this?
a la `# Run this via python -m benchmarks`